src/main/java/model/input/file/DataFile.java

```diff
@@ -18,14 +21,22 @@
  */
 public abstract class DataFile {
 
+	private Logger log = Logger.getLogger("DataFile");
+
+	private String metaDataColumnName;
+	private DataValue metaDataValue;
```

Comment:
Het boeit eigenlijk niet, maar het is wel mooier om hier en in de referenties ernaar toe de generic type <?> erbij te doen.